monitoring/metrics/worker_metrics.py

```
@@ -0,0 +1,108 @@
+# Autonomica Worker Metrics Configuration
+# This file configures Prometheus metrics collection for the worker service
+
+from prometheus_client import Counter, Histogram, Gauge, Summary, generate_latest
```

Ruff (0.12.2)

`prometheus_client.Summary` imported but unused

Remove unused import: `prometheus_client.Summary`

(F401)